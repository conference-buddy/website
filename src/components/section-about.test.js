import React from "react";
import { screen, render, cleanup } from "@testing-library/react";
import AboutSection from "./section-about";

describe("<AboutSection />", () => {
  let component;
  beforeAll(() => {
    component = render(<AboutSection />);
  });

  afterAll(cleanup);

  it("shows a header", () => {
    const header = screen.getByRole("heading", {
      name: "What is Conference Buddy?",
    });

    expect(header).toBeVisible();
  });

  it("shows all elements", () => {
    expect(component.asFragment()).toMatchSnapshot();
  });
});
