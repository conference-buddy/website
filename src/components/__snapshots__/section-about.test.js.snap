// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutSection /> shows all elements 1`] = `
<DocumentFragment>
  <div
    class="container-fluid position-relative bg-bordered-wave"
  >
    <div
      class="p-content-section"
      id="onPageLinkId-section-about"
    >
      <div
        class="container"
      >
        <div
          class="d-none d-sm-block"
        >
          <h2
            class="display-4"
          >
            What is 
            <strong>
              Conference Buddy
            </strong>
            ?
          </h2>
        </div>
        <div
          class="d-block d-sm-none"
        >
          <h2>
            What is Conference Buddy?
          </h2>
        </div>
        <div
          class="row pt-3"
        >
          <div
            class="col-md-6"
          >
            <h3>
              What does it do?
            </h3>
            <p>
              Conference Buddy provides a platform to find 
              <strong>
                companions for tech conferences
              </strong>
              . You can look for one or more person with whom you can attend a conference 
              <strong>
                together
              </strong>
              . Support each other, find a familiar face between all these strangers. You will feel more at ease and comfortable just knowing you’re not alone.
            </p>
            <p>
              At the same time, you have the opportunity to 
              <strong>
                help others
              </strong>
               to feel better! Let’s make tech events more approachable - one Buddy at a time.
            </p>
          </div>
          <div
            class="col-md-6"
          >
            <h3>
              Who can use it?
            </h3>
            <p>
              That's pretty simple: 
              <strong>
                Everyone
              </strong>
               who feels like a bit of support at tech events would help them feel more at ease. At the same time it's for all you great folks out there who want's to be that kind of support!
            </p>
            <p>
              It doesn't matter if it's your first time or if you're an international speaker: You feel nervous or anxious about being around a lot of (stranger) people and interacting with them? Conference Buddy wants to help you 
              <strong>
                feel more comfortable
              </strong>
              !
            </p>
            <p />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
